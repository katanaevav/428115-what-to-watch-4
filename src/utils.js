export const emptyFunction = () => {
  // do nothing
};
