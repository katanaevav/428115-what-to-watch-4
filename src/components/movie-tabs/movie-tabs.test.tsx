import * as React from "react";
import * as renderer from "react-test-renderer";
import MovieTabs from "./movie-tabs";
import {emptyFunction} from "../../utils";


const CURRENT_TAB_INDEX = 0;


it(`Render movie tabs`, () => {
  const tree = renderer.create(
      <MovieTabs
        currentTab = {CURRENT_TAB_INDEX}
        onMovieTabClick = {emptyFunction}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
