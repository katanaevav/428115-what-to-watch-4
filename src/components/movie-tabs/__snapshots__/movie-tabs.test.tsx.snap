// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render movie tabs 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        data-key={0}
        href="#"
        onClick={[Function]}
      >
        Overview
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        data-key={1}
        href="#"
        onClick={[Function]}
      >
        Details
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        data-key={2}
        href="#"
        onClick={[Function]}
      >
        Reviews
      </a>
    </li>
  </ul>
</nav>
`;
